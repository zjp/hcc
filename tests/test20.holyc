"one\n"
"two\a"
"three"
"four
"fi\ve
