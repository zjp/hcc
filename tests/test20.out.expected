STRINGLIT:"one\n" [1,1]
STRINGLIT:"three" [3,1]
EOF [6,1]
